src/components/AdjustPosition/AdjustPosition.tsx

```diff
@@ -450,6 +452,30 @@ const AdjustPosition: FC<AdjustPositionProps> = ({ collateralBalance, debtBalanc
     return newCollateralInDisplayToken.amount.gte(0);
   }, [newCollateralInDisplayToken.amount]);
 
+  const isOverMaxBorrow = useMemo(() => {
+    /**
+     * Do not show error if user did not input anything.
+     */
+    if (!borrowAmount) {
+      return false;
+    }
+
+    if (!protocolStats?.debtSupply || !newDebtTokenValues.amount) {
+      return true;
```

Comment:
Exactly, if some data fails to load, we should not assume that user is not over max debt. But this will not happen, because until protocolStats is loaded, app will show skeleton view. This check is here mostly to prevent TS from complaining.